### PR TITLE
IDENTITY-5177: Fixed Clearing samlssotokenid cookie after SAML2 SSO loging out from IS

### PR DESCRIPTION
**Pull request by GDRDABARERA**
removeTokenIdCookie() wasn't called

### CODE REVIEW

**Comment by stevened04** [comment]
Hello, 
When will this be available? Is it included in a patch or in a new version?

**Comment by darshanasbg** [comment]
Hi @stevened04,

As per [this JIRA](https://wso2.org/jira/browse/IDENTITY-5177), the fix is available with IS 5.3.0 Beta. You can the beta version from [here](https://github.com/wso2/product-is/releases/tag/v5.3.0-beta).

The GA release of IS 5.3.0 planned towards the end of this year.

Regards,

File: components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java [hunk 1]
```diff
@@ -741,6 +742,11 @@ private void handleLogoutResponseFromFramework(HttpServletRequest request,
             SAMLSSOUtil.removeSession(sessionDTO.getSessionId(), validationResponseDTO.getIssuer());
             removeSessionDataFromCache(request.getParameter(SAMLSSOConstants.SESSION_DATA_KEY));
 
+            if ( SSOSessionPersistenceManager.getSessionIndexFromCache(sessionDTO.getSessionId()) == null) {
```
Comment (madurangasiriwardena) [inline components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java]: formatting issue. Remove space after "(".

Comment (GDRDABARERA) [reply components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/servlet/SAMLSSOProviderServlet.java]: Resolved by https://github.com/wso2-extensions/identity-inbound-auth-saml/pull/105

**Pull request merged**